script/scaffold/templates/device_action/integration/__init__.py

```diff
@@ -0,0 +1 @@
+"""Dummy __init__.py to make type checks pass."""
```

Comment:
Please remove this. We copy these files over to the integration directory. Let's not type check the templates.

Reply:
The problem was that pre-commit was running mypy on them, and as these were fatal errors, no `ignore-errors` helped. But excluding them from pre-commit seems to work.